### PR TITLE
Updated the Introduction doc in Requirements Docs

### LINKED ISSUE #468: Bug Report: Introduction page in Desired Features is almost blank
**Describe the bug**
The Introduction page in the Desired Features (further from the Requirement Docs), is blank.

**Expected behavior**
The Introduction page is blank, and should have a proper introduction to all the features by listing out atleast one liners about all them. 

**Actual behavior**
This page feels unnecessary, until there is a summary to all the features and linked up tabs to them.

**Screenshots**
image

**Additional details**
I would like to add the features here, and even create hyperlinks for the user to jump to that specific feature from the Introduction tab to the feature's overview. This would ease in the smooth flow of routing through the documentation.

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by Nitya-Pasrija**
**What kind of change does this PR introduce?**
The Introduction doc page of Requirements Docs is revamped and includes a brief introduction of the Desired Features along with the link to the features for easy navigation of documentation. 

**Issue Number:**
Fixes #468

**Snapshots/Videos:**
image
image
image
image
image

**If relevant, did you update the documentation?**
Yes, previously this page was blank, I added all of this text for a better understanding of the product by the users.

The page updated is: https://docs.talawa.io/docs/features/features-introduction

**Summary**
I've added an Introduction section to make it easier for users to understand the features. It is quite overwhelming to go through a lot of pages to find what one is looking for. With the Introduction section, the users can quickly get an idea of what the features are all about without having to go through everything. I've also included links to related topics so that they can explore more if interested. I did this to make the documentation more user-friendly and convenient for the users.

**Does this PR introduce a breaking change?**
I have updated the documentation to the best of my knowledge and used the Overview pages of the features to write the introduction of all the features. It came to my notice that the "Coordinator Capabilities" features is blank, so I haven't included that in the introduction, but once updated it can be added.

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-docs/blob/develop/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

File: docs/features/introduction.md [hunk 1]
```diff
@@ -3,4 +3,53 @@ id: features-introduction
...
+
+In addition, we plan to integrate automatic group chat creation for events, ensuring that all confirmed attendees can communicate and coordinate effectively. Members of an organization will also be able to send messages to each other through various group chats, promoting collaboration and facilitating communication.
+
+To support effective communication from the organization's leadership, we plan to offer broadcast messaging capabilities, allowing admin and coordinators to send important updates and announcements to all members of the organization. Additionally, we will introduce one-on-one messaging capabilities, enabling users to communicate privately with each other.
+
+As a unique feature, we plan to introduce voice messaging capability with compression, enabling users to communicate more effectively by sending voice messages without consuming excessive bandwidth or storage. By offering these features, we aim to enhance the functionality and user experience of the Chat and Messaging module, promoting collaboration and facilitating communication among users. [Click here to know more](https://docs.talawa.io/docs/features/messaging-and-chat/messaging-and-chat-overview)
+
+
```
Comment (salman-farooqi) [inline docs/features/introduction.md:29]:
```suggestion

```

Comment (Nitya-Pasrija) [reply docs/features/introduction.md:29]: I'm not sure about what you have changed exactly. Can you explain so I can make the necessary changes?

Comment (salman-farooqi) [reply docs/features/introduction.md:29]: > I'm not sure about what you have changed exactly. Can you explain so I can make the necessary changes?

Well, it's not the change in code, it's about the extra line (29) you added to the code.

**Comment by Nitya-Pasrija** [comment]
@anwersayeed @tasneemkoushar @Cioppolo14 
Could you please review this pull request? It needs to be merged before I can create any additional pull requests since I made the changes directly on the "develop" branch of my forked repository instead of creating a new branch.

**Comment by Nitya-Pasrija** [comment]
I created another fork of the project, by deleting the previous one, and used branches now
The updated link to this PR is at: #517 


> @anwersayeed @tasneemkoushar @Cioppolo14 Could you please review this pull request? It needs to be merged before I can create any additional pull requests since I made the changes directly on the "develop" branch of my forked repository instead of creating a new branch.

**Comment by Nitya-Pasrija** [comment]
@anwersayeed @tasneemkoushar @Cioppolo14

**Pull request closed without merging**